scripts/mina-local-network/mina-local-network.sh

```diff
@@ -151,7 +151,7 @@ help() {
                                          |   Default: not set
 -r   |--root                             | When set, override the root working folder (i.e. the value of ROOT) for this script. WARN: this script will clean up anything inside that folder when initializing any run!
                                          |   Default: ${ROOT}
---redirect-logs                          | When set, redirect snark worker logs to file instead of console output
+--redirect-logs                          | When set, redirect all logs (daemons, workers, archive) to file instead of console output
```

Comment:
maybe worth highlight snark worker logs is not controlled by this flag.

Reply:
ok!